.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.hintrc
```

Comment:
- hello, nice job 😉😉 on the project. but there is just a minor error can you pleas remove the hintrc and the stylelint from the gitignore file because it's not recommended to add them inthat file.